markersID: [10,  26,  70, 114, 157, 203, 248, 310, 353, 392, 432, 472, 514, 558, 598, 658, 722, 789, 822, 861, 901, 930, 962, 1002, 1022]
markersXc: [0.5, 1.5, 2.5, 3.5, 4.5, 4.5, 3.5, 2.5, 1.5, 0.5, 0.5, 1.5, 2.5, 3.5, 3.5, 2.5, 1.5, 0.5, 4.5, 4.5, 0.5, 1.5, 2.5, 3.5, 4.5]
markersYc: [0.5, 0.5, 0.5, 0.5, 0.5, 1.5, 1.5, 1.5, 1.5, 1.5, 2.5, 2.5, 2.5, 2.5, 3.5, 3.5, 3.5, 3.5, 2.5, 3.5, 4.5, 4.5, 4.5, 4.5, 4.5]
markersYaw: [-1.5708,  -1.5708,  -1.5708, -1.5708,  -1.5708,  -1.5708, -1.5708,  -1.5708,  -1.5708, -1.5708,  -1.5708,  -1.5708, -1.5708,  -1.5708,  -1.5708, -1.5708,  -1.5708,  -1.5708, -1.5708,  -1.5708,  -1.5708, -1.5708,  -1.5708,  -1.5708, -1.5708]
AxisYawRot: 3.14159265

